### ISSUE TITLE
Incorrect licence abbreviations

### ISSUE THREAD

**Issue by logological** [openjournals/whedon#69]
The JOSS and JOSE LaTeX templates incorrectly abbreviate the Creative Commons Attribution 4.0 International License as "CC-BY".  The [abbreviations preferred by Creative Commons itself](https://creativecommons.org/about/cclicenses/) always put a space, not a hyphen, after the "CC", and separate the "BY", "SA", "ND", and "NC" parts with hyphens.  Their examples of how to apply these licences to one's own works also include the version number of the licence in the abbreviation.

So in short, the occurrences of "CC-BY" in this repository's LaTeX templates should be replaced with "CC BY 4.0".  I will submit a pull request for this.
